src/pages/article-view-page.js

```diff
@@ -72,35 +231,63 @@ class ArticleView extends LitElement{
             
         return html` 
         
-        <navigation-tag></navigation-tag>
         
-        <div class="article-info-container">
-            <div class="article-title-container">
-            <h1>${this.data.article.title}</h1>
-            <user-tag
-                  username=${this.data.article.author.username}
-                  postDate=${this.data.article.updatedAt}
-                  userImg=${this.data.article.author.image}
-                  hearts=${this.data.article.favoritesCount}
-                ></user-tag>
+            <div class="article-info-container">
+                <div class="article-title-container">
+                <h1>${this.data.article.title}</h1>
+                <user-tag
+                    username=${this.data.article.author.username}
+                    postDate=${this.data.article.updatedAt}
+                    userImg=${this.data.article.author.image}
+                    hearts=${this.data.article.favoritesCount}
+                    ></user-tag>
+                </div>
+                
             </div>
-            
-        </div>
-        <div class="article-body-container">
-            <p>${this.data.article.body}</p>
-        </div>
-
-        <div class="comment-section">
-            <textarea-tag
-                placeholder="Write a comment"
-            ></textarea-tag>
-            <btn-tag
-            buttonName="post comment"
-            class="right"
+            <div class="article-body-container">
+                <p>${this.data.article.body}</p>
+            </div>
+
+           ${this.isToken? html` <div class="comment-section">
+                <textarea-tag
+                    placeholder="Write a comment..."
+                    .setValue = ${this.handleChange}
+                    name="comment">
+                </textarea-tag>
+            <div class="belowComment-section">
+            <img src= ${this.userImage||"https://www.w3schools.com/howto/img_avatar.png"} alt="Avatar" class="avatar">
+              <div class="btn-wrapper">  <btn-tag
+                    buttonName="post comment"
+                    .handleClick = ${this.handleSubmit}>
```

Comment:
same here .handleClick

Reply:
we have btn-tag as our own component which takes handleClick as function for two way binding